registrations/tests.py

```diff
@@ -1111,6 +1113,8 @@ def test_friend_only_voice_mon_wed_2_5(self):
         self.assertEqual(d_friend.next_sequence_number, 5)
         self.assertEqual(d_friend.lang, "eng_NG")
         self.assertEqual(d_friend.schedule, 3)
+        self.assertEqual(d_friend.metadata["prepend_next_delivery"],
```

Comment:
I'm actually not sure either. I vaguely recall seeing a note somewhere that for this kind of registration where the same user gets the mother and household messages, that we should send only one welcome message (probably the mother's).  But I can't find it in a scope now, so I'm unsure.